autoPyTorch/evaluation/fit_evaluator.py

```diff
@@ -0,0 +1,317 @@
+import time
```

Comment:
Merge it to `train_evaluator.py`

Reply:
We need it to be separate as we are not gonna use this for search. I can understand making the code more modular so we dont repeat ourselves if we are doing so inappropriately. Maybe you could suggest some improvements